RELEASE.md

```diff
@@ -31,8 +31,10 @@ Maintaining the release branches for older minor releases happens on a best effo
   * kube-state-metrics image tag used in Kubernetes deployment yaml config.
 * Cut the new release branch, i.e., `release-1.2`, or merge/cherry-pick changes onto the minor release branch you intend to tag the release on
 * Cut the new release tag, i.e., `v1.2.0-rc.0`
+* Create a new **pre-release** on github
 * New images are automatically built and pushed to `gcr.io/k8s-staging-kube-state-metrics/kube-state-metrics`
-* Promote image by sending a PR to [kubernetes/k8s.io](https://github.com/kubernetes/k8s.io) repo. Follow the [example PR](https://github.com/kubernetes/k8s.io/pull/1260).
+* Promote image by sending a PR to [kubernetes/k8s.io](https://github.com/kubernetes/k8s.io) repository. Follow the [example PR](https://github.com/kubernetes/k8s.io/pull/1260). Use [cip-mm](https://github.com/kubernetes/release/tree/master/cmd/cip-mm) to update the manifest files in this repository.
+* Create a PR to merge the changes of this release back into to the master branch.
```

Comment:
```suggestion
* Create a PR to merge the changes of this release back into the master/main branch.
```

Reply:
While I agree with the sentiment, I'd keep it as "master" for now since that's what is already mentioned a few lines above in the doc. Looking forward to a name change from master to main if this is already possible on github.com/kubernetes

Reply:
I asked recently when we can rename to main, and they said we will be notified. So far they renamed for a couple of projects but gets tricky when automation is involved.